features/eat-cukes.feature

```diff
@@ -24,14 +24,12 @@ Feature: Eat cukes Example
       [
         {
           "id": "eat-cukes-in-lot",
-          "description": "",
           "name": "Eat cukes in lot",
           "keyword": "Feature",
           "line": 1,
           "elements": [
             {
-              "id": "eat-cukes-in-lot;eating-many-cukes---11",
-              "description": "",
+              "id": "eat-cukes-in-lot;eating-many-cukes",
```

Comment:
I think this actually will be a problem for tools like https://wiki.jenkins.io/display/JENKINS/Cucumber+Living+Documentation+Plugin , as the `id` is not unique anymore. I will check this with one of my projects and let you know about it.

Reply:
Agree. But I can't see where I could break it. Seems I didn't touched anything concerning scenario ID.

Reply:
I'm sorry, but this breaks another plugin we are using. The `id` must be completely unique within the result file.

In the current state, I can not merge this without causing much effort.